### PR TITLE
Don't duplicate output from fastlane now that UI.X is connected

### PR DESCRIPTION
**Pull request by taquitos**

### CODE REVIEW

File: app/shared/models/local_artifact_provider.rb [hunk 1]
```diff
@@ -15,24 +15,21 @@ def default_root_path
       end
     end
 
-    # @return [Pathname]
```
Comment (KrauseFx) [inline app/shared/models/local_artifact_provider.rb]: is that change related?

Comment (taquitos) [reply app/shared/models/local_artifact_provider.rb]: Yeah, I combined them. Will pull this back out though
